profiles/w3c.js

```diff
@@ -28,7 +28,6 @@ const modules = [
   import("../src/core/pluralize.js"),
   import("../src/core/examples.js"),
   import("../src/core/issues-notes.js"),
-  import("../src/core/requirements.js"),
```

Comment:
Could you also remove this from profiles/genovum and profiles/w3c-common?

Reply:
I will remove it for sure. 
Is it the only this making test case fail
Because I am getting error
![Screenshot from 2019-12-23 15-57-04](https://user-images.githubusercontent.com/44131389/71352693-f85a1300-259c-11ea-8cc9-b33b154d21e5.png)
There is no error till your commit of **_use reindent() instead_**
I am still guessing  what happened there.:(
I will fix it out by tonight for sure

Reply:
Maybe you'll want to do `npm run build:w3c` again?